content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -18,10 +18,10 @@ you to figure out what's going wrong.
 ## Running commands in a Pod
 
 For many steps here you will want to see what a Pod running in the cluster
-sees.  The simplest way to do this is to run an interactive alpine Pod:
+sees.  The simplest way to do this is to run an interactive busybox Pod:
 
 ```none
-kubectl run -it --rm --restart=Never alpine --image=alpine sh
+kubectl run -it --rm --restart=Never busybox --image=gcr.io/google-containers/busybox
```

Comment:
better specify the command to run?

Reply:
Yes, added default command to run.

Reply:
LGTM
Not sure if we have a better choice of the image repo though because gcr.io may not be accessible.